GrandCentralBoard/Core/Configuration.swift

```diff
@@ -30,59 +30,59 @@ extension WidgetSettings : Decodable {
 }
 
 public final class ConfigurationDownloader {
-
+    
     private let dataDownloader: DataDownloading
-
+    
     public init(dataDownloader: DataDownloading) {
         self.dataDownloader = dataDownloader
     }
-
+    
     public func fetchConfiguration(fromPath path: String, availableBuilders: [WidgetBuilding], closure: (Result<Configuration>) -> ()) {
         dataDownloader.downloadDataAtPath(path) { result in
             switch result {
-                case .Success(let data):
-                    do {
-                        closure(.Success(try Configuration.configurationFromData(data, availableBuilders: availableBuilders)))
-                    } catch (let error) {
-                        closure(.Failure(error))
-                    }
-                case .Failure(let error):
+            case .Success(let data):
+                do {
+                    closure(.Success(try Configuration.configurationFromData(data, availableBuilders: availableBuilders)))
+                } catch (let error) {
                     closure(.Failure(error))
+                }
+            case .Failure(let error):
+                closure(.Failure(error))
             }
         }
     }
-
+    
 }
 
 public enum ConfigurationError : ErrorType, HavingMessage {
     case WrongFormat
-
+    
     public var message: String {
         switch self {
-            case .WrongFormat:
-                return NSLocalizedString("Wrong format of configuration file!", comment: "")
+        case .WrongFormat:
+            return NSLocalizedString("Wrong format of configuration file!", comment: "")
         }
     }
 }
 
 public struct Configuration {
-
+    
     public let builders: [WidgetBuilding]
     public let settings: [WidgetSettings]
-
+    
     public init(builders: [WidgetBuilding], settings: [WidgetSettings]) {
         self.builders = builders
         self.settings = settings
     }
-
+    
     @warn_unused_result public static func configurationFromData(data: NSData, availableBuilders: [WidgetBuilding]) throws -> Configuration {
-
+        
```

Comment:
Please remove those spaces.

<img width="354" alt="screenshot of xcode 14-03-2016 11-32-09" src="https://cloud.githubusercontent.com/assets/3382607/13741796/64cfa3ba-e9d8-11e5-800e-dd12453590f0.png">